test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,7 +35,19 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
-
+    
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertFalse(Utils.isAnyNull());
+        
+        // null object exist
+        assertTrue(Utils.isAnyNull(null, "abc"));
+        
+        // null object does not exist
+        assertFalse(Utils.isAnyNull("abc", "def"));
+    }
```

Comment:
In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch